service/controller/clusterapi/v31/resource/tccp/template/template_main_record_sets.go

```diff
@@ -67,25 +67,6 @@ const TemplateMainRecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
       Type: A
-  IngressRecordSet:
-    Type: AWS::Route53::RecordSet
-    Properties:
-      AliasTarget:
-        DNSName: !GetAtt IngressLoadBalancer.DNSName
-        HostedZoneId: !GetAtt IngressLoadBalancer.CanonicalHostedZoneNameID
-        EvaluateTargetHealth: false
-      Name: 'ingress.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
-      HostedZoneId: !Ref 'HostedZone'
-      Type: A
-  IngressWildcardRecordSet:
-    Type: AWS::Route53::RecordSet
-    Properties:
-      Name: '*.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
-      HostedZoneId: !Ref 'HostedZone'
-      TTL: '300'
-      Type: CNAME
-      ResourceRecords:
-        - !Ref 'IngressRecordSet'
```

Comment:
Then this needs to be fixed with `'*.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'`
@sslavic fyi

Reply:
Done, PTAL